src/containers/WelcomePage/WelcomePage.jsx

```diff
@@ -163,6 +253,87 @@ export class WelcomePage extends Component {
   }
 }
 
+function mapSearchToFrontPageStructure(data, t) {
+  if (
+    !data ||
+    (data && !data.search) ||
+    (data && data.search && !data.search.results) ||
+    (data &&
+      data.search &&
+      data.search.results &&
+      data.search.results.length === 0)
```

Comment:
Verdt å ta inn optional-chaining bare for dette :D https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining